change-notes/1.20/analysis-javascript.md

```diff
@@ -3,7 +3,8 @@
 ## General improvements
 
 * Support for popular libraries has been improved. Consequently, queries may produce more results on code bases that use the following features:
-  - servers, for example [hapi](https://hapijs.com/)
+  - client-side code, for example [React](https://reactjs.org/)
```

Comment:
Just a note that thanks to [server-side rendering](https://alligator.io/react/server-side-rendering/) it's pretty common that React code runs on the server.

Reply:
Good point, I will keep that in mind.
I will leave this as is though; we will be modifying this file a lot before the next release, and a reorganization of this list is plausible.